src/kernel/emulation/linux/ext/inotify_init.c

```diff
@@ -10,7 +10,11 @@ int inotify_init (void)
 {
 	int rv;
 
-	rv = LINUX_SYSCALL(__NR_inotify_init);
+	#if __NR_inotify_init
+		rv = LINUX_SYSCALL(__NR_inotify_init);
+	#else
+		rv = LINUX_SYSCALL(__NR_inotify_init1, 0);
```

Comment:
Same with this one, [inotify_init1](https://github.com/darlinghq/darling/blob/15c46505cddec16bdf856647f23f5e55db9dec59/src/kernel/emulation/linux/ext/inotify_init1.c#L9) exists.